sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -103,17 +90,13 @@ def configure_azure_monitor(**kwargs) -> None:
     # instanstiated in the other setup steps
     _setup_instrumentations(configurations)
 
-def _setup_resources():
```

Comment:
I think we should keep `setup_resources()` in this file and leave `configurations.py` to be resposible for just populating/defaulting logic. Looks cleaner to me that way.

Reply:
You can do something like `resource = setup_resource()` and then pass the resource into `setup_tracing`,etc.

Reply:
Discussed offline